docs/components/pagination/index.html

```diff
@@ -0,0 +1,144 @@
+---
+title: Pagination
+also:
+  components/buttons: Buttons
+---
+{% extends 'component.njk' %}
+{% block content %}
+  <section>
+    <h2 class="hxSectionTitle" id="first-page">First Page</h2>
+    <div class="demo">
+      <div class="hxPagination hxBtnGroup">
+        <button class="hxBtn" disabled>
+          <hx-icon type="angle-start"></hx-icon>
+        </button>
+        <button class="hxBtn" disabled>
+          <hx-icon type="angle-left"></hx-icon>
+        </button>
+        <button class="hxBtn" aria-current="true">1</button>
+        <button class="hxBtn">2</button>
+        <button class="hxBtn">3</button>
+         <button class="hxBtn">4</button>
+        <button class="hxBtn">5</button>
+        <button class="hxBtn">
+          <hx-icon type="angle-right"></hx-icon>
+        </button>
+        <button class="hxBtn">
+          <hx-icon type="angle-end"></hx-icon>
+        </button>
+      </div>
+    </div>
+    {% code 'html' %}
+      <div class="hxPagination hxBtnGroup">
+        <button class="hxBtn" disabled>
+          <hx-icon type="angle-start"></hx-icon>
+        </button>
+        <button class="hxBtn" disabled>
+          <hx-icon type="angle-left"></hx-icon>
+        </button>
+        <button class="hxBtn" aria-current="true">1</button>
+        <button class="hxBtn">2</button>
+        <button class="hxBtn">3</button>
+        <button class="hxBtn">
+          <hx-icon type="angle-right"></hx-icon>
+        </button>
+        <button class="hxBtn">
+          <hx-icon type="angle-end"></hx-icon>
+        </button>
+      </div>
+    {% endcode %}
+  </section>
+
+  <section>
+    <h2 class="hxSectionTitle" id="second-page">Second Page</h2>
+    <div class="demo">
+      <div class="hxPagination hxBtnGroup">
+        <button class="hxBtn">
+          <hx-icon type="angle-start"></hx-icon>
+        </button>
+        <button class="hxBtn">
+          <hx-icon type="angle-left"></hx-icon>
+        </button>
+        <button class="hxBtn">8</button>
```

Comment:
Are the numbers off here? It starts at 8, but the code sample starts at 1.

Reply:
so for the middle page example we wanted the example to represents single digit and double digit numbers so pagination should start at 8, end at 12, and 10 should be selected